diff -r 2feafdeba393 .hgignore
--- a/.hgignore	Tue Dec 29 08:33:09 2009 +0100
+++ b/.hgignore	Tue Dec 29 09:29:09 2009 +0100
@@ -5,4 +5,4 @@
 .buildpath
 .project
 .settings
-
+nbproject
diff -r 2feafdeba393 modules/os_poker/buddies-invite.tpl.php
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/modules/os_poker/buddies-invite.tpl.php	Tue Dec 29 09:29:09 2009 +0100
@@ -0,0 +1,44 @@
+<?php
+	
+	$dailyChips = 0;
+	$nextDailyChips = 0;
+	$nextBuddyStep = 0;
+
+	if ($current_user)
+	{
+		$task = new CDailyChips();
+		$invites = $current_user->Invites();
+		$nInvites = count($invites["accepted"]);
+		
+		$dailyChips = $task->GetChipsAttributions($nInvites, $nextBuddyStep, $nextDailyChips);
+	}
+
+?>
+
+<div id="buddies-invite">
+   <div class="buddies-invite-left">
+
+   <div class="DailyBonusStep02">
+	   <p class="Dollar1"><?php print 5000; ?></p>
+	   <p class="Dollar2"><?php print $dailyChips; ?></p>
+	   <p class="Dollar3"><?php print $nextDailyChips; ?></p>
+	   <p class="Buddies"><?php print max(($nextBuddyStep - $nInvites), 0); ?></p>
+   </div>
+
+   <p class="Link"><?php print l(t("See present invitations"), "<front>", array("query" => "q=poker/buddies/invitedlist")); ?><br/>
+   <a href="javascript:void(0);" onclick="javascript:parent.tb_remove();"><?php print t("Continue to play"); ?></a>
+   </p>
+
+   <p class="Note">
+   <?php print t("* Please note that just new player are getting this bonus."); ?>
+   </p>
+
+   </div>
+
+   <div class="buddies-invite-right">
+	<?php
+		print($form);
+   ?>
+   </div>
+
+</div>
\ No newline at end of file
diff -r 2feafdeba393 modules/os_poker/buddies-invitedlist.tpl.php
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/modules/os_poker/buddies-invitedlist.tpl.php	Tue Dec 29 09:29:09 2009 +0100
@@ -0,0 +1,150 @@
+<?php
+
+$invites = $current_user->Invites();
+
+$list_invited = array_merge($invites["pending"], $invites["accepted"]);
+usort($list_invited, "_os_poker_sort_invites");
+
+?>
+
+<div id="buddies-invited_list">
+
+ <div id="ContainerContentInvite">
+  
+  <!-- Table Left -->
+  <table cellspacing="0" cellpadding="0" class="TableLeft">
+  <tbody><tr>
+  <th class="Column01">Email</th>
+  <th class="Column02">Date</th>
+  <th class="Column03">Status</th>
+  <th class="Column04">Remind</th>
+  <th style="border-right: medium none;" class="Column05">Delete</th>
+  </tr>
+<?php
+$page = $_GET["page"];
+//print "page : ".$page."<br/>";
+$items_per_page = 15;
+$page_total = ceil(count($list_invited) / $items_per_page) ;
+for ($i = 0; $i < $items_per_page; $i++)
+  {
+	$value = $list_invited[$items_per_page * $page + $i];
+	
+	if ($value)
+    {
+		$list_item = array();
+
+		if ($value->joined != 0)
+		{
+			$invitee = CUserManager::instance()->User($value->invitee);
+			
+			$list_item["name"] = $value->email;
+			$list_item["status"] = 1;
+			$list_item["remind"] = "-";
+			$list_item["delete"] = "-";
+		}
+		else
+		{
+			$list_item["name"] = $value->email;
+			$list_item["status"] = 0;
+			
+			$img = '<img alt="" src="sites/all/themes/poker/images/icon_buttonred.gif"/>';
+			$list_item["remind"] = "<a href='javascript:void(0);' onclick=\"javascript:os_poker_submit(this, 'invite-action-remind-form-{$i}');\">{$img}</a>";
+			$list_item["delete"] = "<a href='javascript:void(0);' onclick=\"javascript:os_poker_submit(this, 'invite-action-delete-form-{$i}');\">{$img}</a>";
+		}
+		
+		$list_item["date"] = date("d.m.y", $value->created);
+  
+?>
+		<tr>	
+			<td class=<?php print '"'.($i % 2 ? "BgWhite" : "BgGrey").'"' ?>><p><?php print $list_item["name"]; ?></p></td>
+			<td class=<?php print '"'.($i % 2 ? "BgWhite" : "BgGrey").'"' ?>><?php print $list_item["date"]; ?></td>
+			<td class=<?php print '"'.($i % 2 ? "BgWhite Center" : "BgGrey Center").'"' ?>><?php print ($list_item["status"] == 1 ? '<img alt="" src="sites/all/themes/poker/images/icon_accepted.gif"/>' : '<img alt="" src="sites/all/themes/poker/images/icon_notconfirmed.gif"/>'); ?></td>
+		
+			<td class=<?php print '"'.($i % 2 ? "BgWhite Center" : "BgGrey Center").'"'?>>
+				<form id="invite-action-remind-form-<?php print $i; ?>" method="post" action="<?php print url("<front>", array("query" => "q=poker/buddies/invitedlist")) ?>" >
+					<input type="hidden" name="invite_target" value="<?php print $value->email; ?>" />
+					<input type="hidden" name="invite_action" value="remind" />
+					<?php print $list_item["remind"]; ?>
+				</form>
+			</td>
+		
+			<td style="border: medium none ;" class=<?php print '"'.($i % 2 ? "BgWhite Center" : "BgGrey Center").'"'?>>
+				<form id="invite-action-delete-form-<?php print $i; ?>" method="post" action="<?php print url("<front>", array("query" => "q=poker/buddies/invitedlist")) ?>" >
+					<input type="hidden" name="invite_target" value="<?php print $value->email; ?>" />
+					<input type="hidden" name="invite_action" value="delete" />
+					<?php print $list_item["delete"]; ?>
+				</form>
+			</td>
+		</tr>
+<?php
+	}
+  }
+?>
+  </tbody>
+ </table>
+
+<div class="PageNavPosition">
+    
+    <!-- Page Navigation -->
+
+    <div class="Clear"></div><div class="PageNav">
+<?php
+$GLOBALS['pager_page_array'][] = $page; //what page you are on
+$GLOBALS['pager_total'][] = $page_total; // total number of pages
+print theme('pager', NULL, $items_per_page);
+?>
+ </div>
+<!--
+Page: <a href="#">1</a> <span class="Marked">2</span> <a href="#">3</a> <a href="#"><span>Â</span></a></div>
+-->
+    <!-- /Page Navigation -->
+
+
+    <!-- Legend -->
+    <div class="Legend">
+    <p>Invitation accepted=</p>
+    <div><img alt="" src="sites/all/themes/poker/images/icon_accepted.gif"/></div>
+    <p style="margin-left: 10px;">Not yet confirmed=</p>
+    <div><img alt="" src="sites/all/themes/poker/images/icon_notconfirmed.gif"/></div>
+    <div class="Clear"/>
+    </div>
+    <!-- /Legend -->
+    
+    </div>
+</div>
+
+
+
+<table cellspacing="0" cellpadding="0" class="TableRight">
+    <tbody><tr>
+    <th colspan="2">Overview</th>
+    </tr>
+    
+    <tr>
+    <td colspan="2">You have invited<br/><span><?php print $invites["total"]; ?></span> friends</td>
+    </tr>
+    <tr><td colspan="2" class="Dotted"><div class="LineDotted"/></td></tr>
+    
+    <tr>
+    <td>Accepted<br/>invitations:</td>
+    <td class="Number"><br/><span><?php print count($invites["accepted"]); ?></span></td>
+    </tr>
+    <tr><td colspan="2" class="Dotted"><div class="LineDotted"/></td></tr>
+
+    <tr>
+    <td>Pending<br/>invitations:</td>
+    <td class="Number"><br/><span><?php print count($invites["pending"]); ?></span></td>
+    </tr>
+    
+    </tbody>
+</table>
+
+<div class="InviteMore">
+    <a href="?q=poker/buddies/invite"><img src="sites/all/themes/poker/images/banner_invite_more_friends.jpg"/></a>
+    </div>
+
+ </div>
+
+
+
+</div>
\ No newline at end of file
diff -r 2feafdeba393 modules/os_poker/buddies-list.tpl.php
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/modules/os_poker/buddies-list.tpl.php	Tue Dec 29 09:29:09 2009 +0100
@@ -0,0 +1,138 @@
+<div id="buddies-list">
+<?php	if ($buddies && count($buddies) > 0) : ?>
+
+	<?php
+$page = $_GET["page"];
+$items_per_page = 8;
+$page_total = (int)(count($buddies) / $items_per_page);
+
+if (count($buddies) % $items_per_page)
+	++$page_total;
+
+for ($i = 0; $i < $items_per_page; $i++)
+  {
+  $buddies_entry = $buddies[$items_per_page * $page + $i];
+  if ($buddies_entry)
+    /*	foreach ($buddies as $buddies_entry)*/
+    {
+		$oBuddy->name = $buddies_entry->profile_nickname;
+		$oBuddy->uid =  $buddies_entry->uid;
+						
+	?>
+		<div class='buddy_result_list_entry'>
+
+			<div class="buddy_result_list_picture">
+			<?php
+				if (variable_get('user_relationships_show_user_pictures', 0))
+				{
+					$alt = t("@user's picture", array('@user' => $buddies_entry->name ? $buddies_entry->name : variable_get('anonymous', t('Anonymous'))));
+			  
+					$picture = theme('image', $buddies_entry->picture, $alt, $alt, '', FALSE);
+					$attributes = array('attributes' => array('title' => t('View user profile.')), 'html' => TRUE);
+					print l($picture, "poker/profile/profile/$oBuddy->uid", $attributes);
+				}
+			?>
+			</div>
+
+			<div class="buddy_result_list_infos">
+				<div class="buddy_result_list_name <?php if ($buddies_entry->Online()) { print "online"; } ?>">
+					<?php 
+						print theme('username', $oBuddy);
+					?>
+				</div>
+				<div class="buddy_result_list_chips">
+					<?php print $buddies_entry->Chips(TRUE); ?>
+				</div>
+				<div class="buddy_result_list_level">
+					<?php print $buddies_entry->Status(); ?>
+				</div>
+				<div class="buddy_result_list_city">
+					<?php print $buddies_entry->profile_city; ?>
+				</div>
+			</div>
+			<div class="buddy_result_list_links">
+				<div class="link_challenge poker_submit silver" onclick="javascript:os_poker_send_message({type :'os_poker_challenge_user', challengetarget: <?php print $buddies_entry->uid; ?>});">
+					<div class="pre"> </div>
+					<div class="label" style="width: 50px; text-align: center;">
+						<?php print t("Challenge"); ?>
+					</div>
+				</div>
+				
+				<?php
+
+					if ($buddies_entry->Online())
+					{
+						$text = t("Join table");
+						$url = "os_poker_send_message({type :'os_poker_invite_user', target: " . $buddies_entry->uid . ", online: true})";
+					}
+					else
+					{
+						$text = t("Invite now");
+						$url = "os_poker_send_message({type :'os_poker_invite_user', target: " . $buddies_entry->uid . "})";
+					}
+					
+				?>
+				<div class="link_invite poker_submit silver" onclick="javascript:<?php print $url; ?>;">
+					<div class="pre"> </div>
+					<div class="label" style="width: 50px; text-align: center;">
+						<?php
+							print $text;
+						?>
+					</div>
+				</div>
+				<?php if ($buddies_entry->profile_accept_gifts == 0) { ?>
+					<div class="link_send_gifts poker_submit silver" onclick="javascript: document.location.href ='<?php print url("<front>", array("query" => "q=poker/shop/shop/1/buddy/" . $buddies_entry->uid)); ?>';"  >
+						<div class="pre"> </div>
+						<div class="label" style="width: 50px; text-align: center;">
+							<?php print t("Send gifts"); ?>
+						</div>
+					</div>
+				<?php } ?>
+			</div>
+		</div>
+	<?
+	}
+  }
+?>
+<div class="BannerTop">
+  <?php if ($current_user->CanDailyGift()) { ?>
+								<img style="cursor: pointer;" onclick="javascript:os_poker_send_message({type: 'os_poker_daily_gift'}); this.style.cursor=''; this.src='<?php  print drupal_get_path('theme', 'poker'); ?>/images/banner_invite_more_friends.jpg';" src="<?php  print drupal_get_path('theme', 'poker'); ?>/images/banner_free_gifts.jpg" alt="Today's free gift"/>
+		     <?php } else { ?>
+			       <img src="<?php  print drupal_get_path('theme', 'poker'); ?>/images/banner_invite_more_friends.jpg" alt="Today's free gift"/>
+		     <?php } ?>
+
+</div>
+<div class="BannerBottom">
+ <a title="" href="?q=poker/buddies/invite">
+  <img alt="" src="sites/all/themes/poker/images/banner_invite_more_friends.jpg"/>
+ </a>
+</div>
+
+<div class="clear"></div>
+
+<div class="footer">
+<?php
+$GLOBALS['pager_page_array'][] = $page; //what page you are on
+$GLOBALS['pager_total'][] = $page_total; // total number of pages
+print theme('pager', NULL, $items_per_page);
+?>
+</div>
+
+<?php else : ?>
+	<?php
+		if ($action == "search")
+		{
+		  $error_message = "<h1>" . t("Sorry !") . "</h1>" . t("your Searchrequest didn't match any Players.") . "<br/><br/>". "<div class='poker_submit'><div class='pre'>&nbsp;</div><div class='label'>" . l(t("Try again"), 'poker/buddies/search') . "</div></div>";
+
+
+		}
+		else
+		{
+			$error_message = t("You don't have any buddy.") . "<br/>". l(t("Invite a buddy"), 'poker/buddies/invite');
+		}
+		
+		print theme('poker_error_message', $error_message);
+	?>
+<?php endif; ?> 
+
+</div>
\ No newline at end of file
diff -r 2feafdeba393 modules/os_poker/buddies-search.tpl.php
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/modules/os_poker/buddies-search.tpl.php	Tue Dec 29 09:29:09 2009 +0100
@@ -0,0 +1,11 @@
+<div id="buddies-search">
+   <div class="buddies-search-banner">
+   <h2><?php print t("Looking for other players?"); ?></h2>
+   <p><?php print t("Then you've come to the right place! Whether you're looking for someone nearby, nationwide or in a foreign country, we show you all Poker players at a glance."); ?></p>
+   </div>
+	<?php print($form); ?>
+
+   <p class="buddies-search-infos">
+   <?php print t("* Please note that you can only add players to your buddy list or send gifts when you are both playing at the same table."); ?>
+   </p>
+</div>
\ No newline at end of file
diff -r 2feafdeba393 modules/os_poker/buddies-tabs.tpl.php
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/modules/os_poker/buddies-tabs.tpl.php	Tue Dec 29 09:29:09 2009 +0100
@@ -0,0 +1,8 @@
+<a class="LayerClose" onclick="javascript:parent.tb_remove();" href="javascript:void(0);">&nbsp;</a>
+<div class="tabs">
+	<ul class="tabs primary">
+		<li <?php if ($action == NULL) { print 'class="active"'; } ?>><a href="?q=poker/buddies" ><?php print "Buddies"; ?></a></li>
+		<li <?php if ($action == "search") { print 'class="active"'; } ?>><a href="?q=poker/buddies/search"><?php print "Search"; ?></a></li>
+		<li <?php if ($action == "invite" || $action == "invitedlist") { print 'class="active"'; } ?>><a href="?q=poker/buddies/invite"><?php print "Invite friends"; ?></a></li>
+	</ul>
+</div>
diff -r 2feafdeba393 modules/os_poker/buddy-block.tpl.php
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/modules/os_poker/buddy-block.tpl.php	Tue Dec 29 09:29:09 2009 +0100
@@ -0,0 +1,78 @@
+<div class="buddy_list_block <?php if ($buddy && $buddy->Online()) { print "online"; } ?>">
+
+	<div class="buddy_list_number <?php if ($buddy && $buddyNumber == 1) print "golden"; ?>">
+		<?php print $buddyNumber; ?>
+	</div>
+	
+	<?php if ($buddy) { ?>
+
+		<div class="buddy_list_name <?php if ($buddy->Online()) { print "online"; } ?>">
+			<?php 
+				$oBuddy->name = $buddy->profile_nickname;
+				$oBuddy->uid =  $buddy->uid;
+				
+				print theme('username', $oBuddy);
+			?>
+		</div>
+		<div class="buddy_list_picture">
+			<?php
+			if (variable_get('user_relationships_show_user_pictures', 0))
+			{
+				$alt = t("@user's picture", array('@user' => $buddy->name ? $buddy->name : variable_get('anonymous', t('Anonymous'))));
+		  
+				$picture = theme('image', $buddy->picture, $alt, $alt, '', FALSE);
+				$attributes = array('attributes' => array('title' => t('View user profile.'), "class" => "thickbox"),
+									"query" => array("height" => 442,
+													"width" => 603,
+													"TB_iframe" => "true"),
+									'html' => TRUE);
+				print l($picture, "poker/profile/profile/$oBuddy->uid", $attributes);
+			}
+			?>
+		</div>
+		<div class="buddy_list_chips">
+			<?php print $buddy->Chips(TRUE); ?>
+		</div>
+		<?php if (!$hide_links) { ?>
+		<div class="buddy_list_links">
+			<a href="javascript:void(0);" onclick="javascript:os_poker_send_message({type :'os_poker_challenge_user', challengetarget: <?php print $buddy->uid; ?>});"><?php print t("Challenge"); ?>&nbsp;&gt;&gt;</a><br/>
+			<?php
+			
+				if ($buddy->Online())
+				{
+					$text = t("Join table");
+					$url = "os_poker_send_message({type :'os_poker_invite_user', target: " . $buddy->uid . ", online: true})";
+				}
+				else
+				{
+					$text = t("Invite now");
+					$url = "os_poker_send_message({type :'os_poker_invite_user', target: " . $buddy->uid . "})";
+				}
+			?>
+			<a href="javascript:void(0);" onclick="javascript:<?php print $url; ?>;"><?php print $text; ?>&nbsp;&gt;&gt;</a><br/>
+			<?php
+				if ($buddy->profile_accept_gifts == 0)
+				{
+					print l(t("Send Gifts") . "&nbsp;&gt;&gt;", "poker/shop/shop/1/buddy/" . $buddy->uid, array("query" => array("height" => 442,
+																																 "width" => 603,
+																																 "TB_iframe" => "true"),
+																												"attributes" => array("class" => "thickbox"),					
+																												"html" => TRUE));
+				}
+			?>
+		</div>
+		<?php } ?>
+	<?php } else { ?>
+		<div class="buddy_list_placeholder">
+			<div class="title">
+				<?php print t("Invite"); ?>
+			</div>
+			<div class="body">
+				<?php print l(t("Invite Friends"), "poker/buddies/invite", array("query" => array(	"height" => 442,
+																									"width" => 603,
+																									"TB_iframe" => "true"),
+																				"attributes" => array("class" => "thickbox"))); ?>
+			</div>
+		</div>
+	<?php } ?>
+</div>
diff -r 2feafdeba393 modules/os_poker/buddy-list.tpl.php
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/modules/os_poker/buddy-list.tpl.php	Tue Dec 29 09:29:09 2009 +0100
@@ -0,0 +1,75 @@
+<div id="buddylist">
+	<div class="header block_title_bar block_title_text">
+		<div class="fleft">
+			<?php print $title; ?>
+		</div>
+		<div class="fleft">
+			<?php print $filter_form; ?>
+		</div>
+		<div class="clear"></div>
+	</div>
+
+
+
+	<div id="buddy_panel">
+
+		<a href="javascript:void(0);" class="previous">&nbsp;</a>
+		<div class="list">
+			<?php
+				$limit = 0;
+			
+				if ($current_user)
+				{
+					$buddies = $current_user->Buddies(TRUE);
+					$rcount = count($buddies);
+					$limit = $rcount;
+					$placeholder = FALSE;
+					if ($rcount < 7)
+					{
+						$limit = 7;
+						$placeholder = TRUE;
+					}
+				}
+				
+			?>
+			<div class="cursor">
+				<?php 
+					for ($i = 0; $i < $limit; ++$i)
+					{
+						if ($placeholder && $i >= $rcount)
+						{
+							print theme('buddy_block', NULL, $i + 1);
+						}
+						else
+						{
+							$buddy = &$buddies[$i];
+							print theme('buddy_block', $buddy, $i + 1);
+						}
+					}
+				?>
+			</div>
+		</div>
+		<a href="javascript:void(0);" class="next">&nbsp;</a>
+
+
+		<div id="today_gift">
+		     <?php if ($current_user && $current_user->CanDailyGift()) { ?>
+								<img width="115px" height="165px" style="cursor: pointer;" onclick="javascript: os_poker_switch_today_gift_blocks();  os_poker_send_message({type: 'os_poker_daily_gift'}); this.style.cursor='';" src="<?php  print drupal_get_path('theme', 'poker'); ?>/images/daily_gift.png" alt="Today's free gift"/>
+		     <?php } else { ?>
+                                  <a class="thickbox" title="" href="/drupal6/?q=poker/buddies/invite/&height=442&width=603&keepThis=true&TB_iframe=true">
+                          <img width="112px" height="166px" src="<?php  print drupal_get_path('theme', 'poker'); ?>/images/banner_invite_more_friends.jpg" alt="Today's free gift"/>
+                                    </a>
+		     <?php } ?>
+		</div>
+		     <div id="today_gift_invite">
+                                 <a class="thickbox" title="" href="/drupal6/?q=poker/buddies/invite/&height=442&width=603&keepThis=true&TB_iframe=true">
+                          <img width="112px" height="166px" src="<?php  print drupal_get_path('theme', 'poker'); ?>/images/banner_invite_more_friends.jpg" alt="Today's free gift"/>
+                                    </a>
+		     </div>
+
+
+		<div id="invite_friends">
+		</div>
+		<div class="clear"></div>
+	</div>
+</div>
diff -r 2feafdeba393 modules/os_poker/os_poker.module
--- a/modules/os_poker/os_poker.module	Tue Dec 29 08:33:09 2009 +0100
+++ b/modules/os_poker/os_poker.module	Tue Dec 29 09:29:09 2009 +0100
@@ -383,9 +383,35 @@
 												'arguments' => array(),
 												'template' => 'footer',
 											),
-				);
-
-
+    'buddy_block' => array(
+      'arguments' => array('buddy' => NULL, 'buddyNumber' => NULL, 'hide_links' => FALSE),
+      'template'  => 'buddy-block',
+    ),
+    'buddies_tabs' => array(
+      'arguments' => array('action' => NULL),
+      'template'  => 'buddies-tabs',
+    ),
+    'buddies_list' => array(
+      'arguments' => array('buddies' => NULL, 'current_user' => NULL, 'action' => NULL, 'page' => NULL),
+      'template'  => 'buddies-list',
+    ),
+    'buddies_invite' => array(
+      'arguments' => array('form' => NULL, 'current_user' => NULL),
+      'template'  => 'buddies-invite',
+    ),
+    'buddies_invitedlist' => array(
+      'arguments' => array('current_user' => NULL),
+      'template'  => 'buddies-invitedlist',
+    ),
+    'buddies_search' => array(
+      'arguments' => array('form' => NULL, 'current_user' => NULL),
+      'template'  => 'buddies-search',
+    ),
+    'poker_error_message' => array(
+      'arguments' => array('text' => NULL),
+      'template'  => 'poker-error-message',
+    ),
+  );
 }
 
 /**
diff -r 2feafdeba393 modules/os_poker/poker-error-message.tpl.php
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/modules/os_poker/poker-error-message.tpl.php	Tue Dec 29 09:29:09 2009 +0100
@@ -0,0 +1,5 @@
+<div class="poker_error_message">
+		<div class="msg">
+			<?php print $text; ?>
+		</div>
+</div>
\ No newline at end of file
diff -r 2feafdeba393 themes/poker/buddies_invite.tpl.php
--- a/themes/poker/buddies_invite.tpl.php	Tue Dec 29 08:33:09 2009 +0100
+++ /dev/null	Thu Jan 01 00:00:00 1970 +0000
@@ -1,44 +0,0 @@
-<?php
-	
-	$dailyChips = 0;
-	$nextDailyChips = 0;
-	$nextBuddyStep = 0;
-
-	if ($current_user)
-	{
-		$task = new CDailyChips();
-		$invites = $current_user->Invites();
-		$nInvites = count($invites["accepted"]);
-		
-		$dailyChips = $task->GetChipsAttributions($nInvites, $nextBuddyStep, $nextDailyChips);
-	}
-
-?>
-
-<div id="buddies-invite">
-   <div class="buddies-invite-left">
-
-   <div class="DailyBonusStep02">
-	   <p class="Dollar1"><?php print 5000; ?></p>
-	   <p class="Dollar2"><?php print $dailyChips; ?></p>
-	   <p class="Dollar3"><?php print $nextDailyChips; ?></p>
-	   <p class="Buddies"><?php print max(($nextBuddyStep - $nInvites), 0); ?></p>
-   </div>
-
-   <p class="Link"><?php print l(t("See present invitations"), "<front>", array("query" => "q=poker/buddies/invitedlist")); ?><br/>
-   <a href="javascript:void(0);" onclick="javascript:parent.tb_remove();"><?php print t("Continue to play"); ?></a>
-   </p>
-
-   <p class="Note">
-   <?php print t("* Please note that just new player are getting this bonus."); ?>
-   </p>
-
-   </div>
-
-   <div class="buddies-invite-right">
-	<?php
-		print($form);
-   ?>
-   </div>
-
-</div>
\ No newline at end of file
diff -r 2feafdeba393 themes/poker/buddies_invitedlist.tpl.php
--- a/themes/poker/buddies_invitedlist.tpl.php	Tue Dec 29 08:33:09 2009 +0100
+++ /dev/null	Thu Jan 01 00:00:00 1970 +0000
@@ -1,150 +0,0 @@
-<?php
-
-$invites = $current_user->Invites();
-
-$list_invited = array_merge($invites["pending"], $invites["accepted"]);
-usort($list_invited, "_os_poker_sort_invites");
-
-?>
-
-<div id="buddies-invited_list">
-
- <div id="ContainerContentInvite">
-  
-  <!-- Table Left -->
-  <table cellspacing="0" cellpadding="0" class="TableLeft">
-  <tbody><tr>
-  <th class="Column01">Email</th>
-  <th class="Column02">Date</th>
-  <th class="Column03">Status</th>
-  <th class="Column04">Remind</th>
-  <th style="border-right: medium none;" class="Column05">Delete</th>
-  </tr>
-<?php
-$page = $_GET["page"];
-//print "page : ".$page."<br/>";
-$items_per_page = 15;
-$page_total = ceil(count($list_invited) / $items_per_page) ;
-for ($i = 0; $i < $items_per_page; $i++)
-  {
-	$value = $list_invited[$items_per_page * $page + $i];
-	
-	if ($value)
-    {
-		$list_item = array();
-
-		if ($value->joined != 0)
-		{
-			$invitee = CUserManager::instance()->User($value->invitee);
-			
-			$list_item["name"] = $value->email;
-			$list_item["status"] = 1;
-			$list_item["remind"] = "-";
-			$list_item["delete"] = "-";
-		}
-		else
-		{
-			$list_item["name"] = $value->email;
-			$list_item["status"] = 0;
-			
-			$img = '<img alt="" src="sites/all/themes/poker/images/icon_buttonred.gif"/>';
-			$list_item["remind"] = "<a href='javascript:void(0);' onclick=\"javascript:os_poker_submit(this, 'invite-action-remind-form-{$i}');\">{$img}</a>";
-			$list_item["delete"] = "<a href='javascript:void(0);' onclick=\"javascript:os_poker_submit(this, 'invite-action-delete-form-{$i}');\">{$img}</a>";
-		}
-		
-		$list_item["date"] = date("d.m.y", $value->created);
-  
-?>
-		<tr>	
-			<td class=<?php print '"'.($i % 2 ? "BgWhite" : "BgGrey").'"' ?>><p><?php print $list_item["name"]; ?></p></td>
-			<td class=<?php print '"'.($i % 2 ? "BgWhite" : "BgGrey").'"' ?>><?php print $list_item["date"]; ?></td>
-			<td class=<?php print '"'.($i % 2 ? "BgWhite Center" : "BgGrey Center").'"' ?>><?php print ($list_item["status"] == 1 ? '<img alt="" src="sites/all/themes/poker/images/icon_accepted.gif"/>' : '<img alt="" src="sites/all/themes/poker/images/icon_notconfirmed.gif"/>'); ?></td>
-		
-			<td class=<?php print '"'.($i % 2 ? "BgWhite Center" : "BgGrey Center").'"'?>>
-				<form id="invite-action-remind-form-<?php print $i; ?>" method="post" action="<?php print url("<front>", array("query" => "q=poker/buddies/invitedlist")) ?>" >
-					<input type="hidden" name="invite_target" value="<?php print $value->email; ?>" />
-					<input type="hidden" name="invite_action" value="remind" />
-					<?php print $list_item["remind"]; ?>
-				</form>
-			</td>
-		
-			<td style="border: medium none ;" class=<?php print '"'.($i % 2 ? "BgWhite Center" : "BgGrey Center").'"'?>>
-				<form id="invite-action-delete-form-<?php print $i; ?>" method="post" action="<?php print url("<front>", array("query" => "q=poker/buddies/invitedlist")) ?>" >
-					<input type="hidden" name="invite_target" value="<?php print $value->email; ?>" />
-					<input type="hidden" name="invite_action" value="delete" />
-					<?php print $list_item["delete"]; ?>
-				</form>
-			</td>
-		</tr>
-<?php
-	}
-  }
-?>
-  </tbody>
- </table>
-
-<div class="PageNavPosition">
-    
-    <!-- Page Navigation -->
-
-    <div class="Clear"></div><div class="PageNav">
-<?php
-$GLOBALS['pager_page_array'][] = $page; //what page you are on
-$GLOBALS['pager_total'][] = $page_total; // total number of pages
-print theme('pager', NULL, $items_per_page);
-?>
- </div>
-<!--
-Page: <a href="#">1</a> <span class="Marked">2</span> <a href="#">3</a> <a href="#"><span>Â</span></a></div>
--->
-    <!-- /Page Navigation -->
-
-
-    <!-- Legend -->
-    <div class="Legend">
-    <p>Invitation accepted=</p>
-    <div><img alt="" src="sites/all/themes/poker/images/icon_accepted.gif"/></div>
-    <p style="margin-left: 10px;">Not yet confirmed=</p>
-    <div><img alt="" src="sites/all/themes/poker/images/icon_notconfirmed.gif"/></div>
-    <div class="Clear"/>
-    </div>
-    <!-- /Legend -->
-    
-    </div>
-</div>
-
-
-
-<table cellspacing="0" cellpadding="0" class="TableRight">
-    <tbody><tr>
-    <th colspan="2">Overview</th>
-    </tr>
-    
-    <tr>
-    <td colspan="2">You have invited<br/><span><?php print $invites["total"]; ?></span> friends</td>
-    </tr>
-    <tr><td colspan="2" class="Dotted"><div class="LineDotted"/></td></tr>
-    
-    <tr>
-    <td>Accepted<br/>invitations:</td>
-    <td class="Number"><br/><span><?php print count($invites["accepted"]); ?></span></td>
-    </tr>
-    <tr><td colspan="2" class="Dotted"><div class="LineDotted"/></td></tr>
-
-    <tr>
-    <td>Pending<br/>invitations:</td>
-    <td class="Number"><br/><span><?php print count($invites["pending"]); ?></span></td>
-    </tr>
-    
-    </tbody>
-</table>
-
-<div class="InviteMore">
-    <a href="?q=poker/buddies/invite"><img src="sites/all/themes/poker/images/banner_invite_more_friends.jpg"/></a>
-    </div>
-
- </div>
-
-
-
-</div>
\ No newline at end of file
diff -r 2feafdeba393 themes/poker/buddies_list.tpl.php
--- a/themes/poker/buddies_list.tpl.php	Tue Dec 29 08:33:09 2009 +0100
+++ /dev/null	Thu Jan 01 00:00:00 1970 +0000
@@ -1,138 +0,0 @@
-<div id="buddies-list">
-<?php	if ($buddies && count($buddies) > 0) : ?>
-
-	<?php
-$page = $_GET["page"];
-$items_per_page = 8;
-$page_total = (int)(count($buddies) / $items_per_page);
-
-if (count($buddies) % $items_per_page)
-	++$page_total;
-
-for ($i = 0; $i < $items_per_page; $i++)
-  {
-  $buddies_entry = $buddies[$items_per_page * $page + $i];
-  if ($buddies_entry)
-    /*	foreach ($buddies as $buddies_entry)*/
-    {
-		$oBuddy->name = $buddies_entry->profile_nickname;
-		$oBuddy->uid =  $buddies_entry->uid;
-						
-	?>
-		<div class='buddy_result_list_entry'>
-
-			<div class="buddy_result_list_picture">
-			<?php
-				if (variable_get('user_relationships_show_user_pictures', 0))
-				{
-					$alt = t("@user's picture", array('@user' => $buddies_entry->name ? $buddies_entry->name : variable_get('anonymous', t('Anonymous'))));
-			  
-					$picture = theme('image', $buddies_entry->picture, $alt, $alt, '', FALSE);
-					$attributes = array('attributes' => array('title' => t('View user profile.')), 'html' => TRUE);
-					print l($picture, "poker/profile/profile/$oBuddy->uid", $attributes);
-				}
-			?>
-			</div>
-
-			<div class="buddy_result_list_infos">
-				<div class="buddy_result_list_name <?php if ($buddies_entry->Online()) { print "online"; } ?>">
-					<?php 
-						print theme('username', $oBuddy);
-					?>
-				</div>
-				<div class="buddy_result_list_chips">
-					<?php print $buddies_entry->Chips(TRUE); ?>
-				</div>
-				<div class="buddy_result_list_level">
-					<?php print $buddies_entry->Status(); ?>
-				</div>
-				<div class="buddy_result_list_city">
-					<?php print $buddies_entry->profile_city; ?>
-				</div>
-			</div>
-			<div class="buddy_result_list_links">
-				<div class="link_challenge poker_submit silver" onclick="javascript:os_poker_send_message({type :'os_poker_challenge_user', challengetarget: <?php print $buddies_entry->uid; ?>});">
-					<div class="pre"> </div>
-					<div class="label" style="width: 50px; text-align: center;">
-						<?php print t("Challenge"); ?>
-					</div>
-				</div>
-				
-				<?php
-
-					if ($buddies_entry->Online())
-					{
-						$text = t("Join table");
-						$url = "os_poker_send_message({type :'os_poker_invite_user', target: " . $buddies_entry->uid . ", online: true})";
-					}
-					else
-					{
-						$text = t("Invite now");
-						$url = "os_poker_send_message({type :'os_poker_invite_user', target: " . $buddies_entry->uid . "})";
-					}
-					
-				?>
-				<div class="link_invite poker_submit silver" onclick="javascript:<?php print $url; ?>;">
-					<div class="pre"> </div>
-					<div class="label" style="width: 50px; text-align: center;">
-						<?php
-							print $text;
-						?>
-					</div>
-				</div>
-				<?php if ($buddies_entry->profile_accept_gifts == 0) { ?>
-					<div class="link_send_gifts poker_submit silver" onclick="javascript: document.location.href ='<?php print url("<front>", array("query" => "q=poker/shop/shop/1/buddy/" . $buddies_entry->uid)); ?>';"  >
-						<div class="pre"> </div>
-						<div class="label" style="width: 50px; text-align: center;">
-							<?php print t("Send gifts"); ?>
-						</div>
-					</div>
-				<?php } ?>
-			</div>
-		</div>
-	<?
-	}
-  }
-?>
-<div class="BannerTop">
-  <?php if ($current_user->CanDailyGift()) { ?>
-								<img style="cursor: pointer;" onclick="javascript:os_poker_send_message({type: 'os_poker_daily_gift'}); this.style.cursor=''; this.src='<?php  print drupal_get_path('theme', 'poker'); ?>/images/banner_invite_more_friends.jpg';" src="<?php  print drupal_get_path('theme', 'poker'); ?>/images/banner_free_gifts.jpg" alt="Today's free gift"/>
-		     <?php } else { ?>
-			       <img src="<?php  print drupal_get_path('theme', 'poker'); ?>/images/banner_invite_more_friends.jpg" alt="Today's free gift"/>
-		     <?php } ?>
-
-</div>
-<div class="BannerBottom">
- <a title="" href="?q=poker/buddies/invite">
-  <img alt="" src="sites/all/themes/poker/images/banner_invite_more_friends.jpg"/>
- </a>
-</div>
-
-<div class="clear"></div>
-
-<div class="footer">
-<?php
-$GLOBALS['pager_page_array'][] = $page; //what page you are on
-$GLOBALS['pager_total'][] = $page_total; // total number of pages
-print theme('pager', NULL, $items_per_page);
-?>
-</div>
-
-<?php else : ?>
-	<?php
-		if ($action == "search")
-		{
-		  $error_message = "<h1>" . t("Sorry !") . "</h1>" . t("your Searchrequest didn't match any Players.") . "<br/><br/>". "<div class='poker_submit'><div class='pre'>&nbsp;</div><div class='label'>" . l(t("Try again"), 'poker/buddies/search') . "</div></div>";
-
-
-		}
-		else
-		{
-			$error_message = t("You don't have any buddy.") . "<br/>". l(t("Invite a buddy"), 'poker/buddies/invite');
-		}
-		
-		print theme('poker_error_message', $error_message);
-	?>
-<?php endif; ?> 
-
-</div>
\ No newline at end of file
diff -r 2feafdeba393 themes/poker/buddies_search.tpl.php
--- a/themes/poker/buddies_search.tpl.php	Tue Dec 29 08:33:09 2009 +0100
+++ /dev/null	Thu Jan 01 00:00:00 1970 +0000
@@ -1,11 +0,0 @@
-<div id="buddies-search">
-   <div class="buddies-search-banner">
-   <h2><?php print t("Looking for other players?"); ?></h2>
-   <p><?php print t("Then you've come to the right place! Whether you're looking for someone nearby, nationwide or in a foreign country, we show you all Poker players at a glance."); ?></p>
-   </div>
-	<?php print($form); ?>
-
-   <p class="buddies-search-infos">
-   <?php print t("* Please note that you can only add players to your buddy list or send gifts when you are both playing at the same table."); ?>
-   </p>
-</div>
\ No newline at end of file
diff -r 2feafdeba393 themes/poker/buddies_tabs.tpl.php
--- a/themes/poker/buddies_tabs.tpl.php	Tue Dec 29 08:33:09 2009 +0100
+++ /dev/null	Thu Jan 01 00:00:00 1970 +0000
@@ -1,8 +0,0 @@
-<a class="LayerClose" onclick="javascript:parent.tb_remove();" href="javascript:void(0);">&nbsp;</a>
-<div class="tabs">
-	<ul class="tabs primary">
-		<li <?php if ($action == NULL) { print 'class="active"'; } ?>><a href="?q=poker/buddies" ><?php print "Buddies"; ?></a></li>
-		<li <?php if ($action == "search") { print 'class="active"'; } ?>><a href="?q=poker/buddies/search"><?php print "Search"; ?></a></li>
-		<li <?php if ($action == "invite" || $action == "invitedlist") { print 'class="active"'; } ?>><a href="?q=poker/buddies/invite"><?php print "Invite friends"; ?></a></li>
-	</ul>
-</div>
diff -r 2feafdeba393 themes/poker/buddy_list_block.tpl.php
--- a/themes/poker/buddy_list_block.tpl.php	Tue Dec 29 08:33:09 2009 +0100
+++ /dev/null	Thu Jan 01 00:00:00 1970 +0000
@@ -1,78 +0,0 @@
-<div class="buddy_list_block <?php if ($buddy && $buddy->Online()) { print "online"; } ?>">
-
-	<div class="buddy_list_number <?php if ($buddy && $buddyNumber == 1) print "golden"; ?>">
-		<?php print $buddyNumber; ?>
-	</div>
-	
-	<?php if ($buddy) { ?>
-
-		<div class="buddy_list_name <?php if ($buddy->Online()) { print "online"; } ?>">
-			<?php 
-				$oBuddy->name = $buddy->profile_nickname;
-				$oBuddy->uid =  $buddy->uid;
-				
-				print theme('username', $oBuddy);
-			?>
-		</div>
-		<div class="buddy_list_picture">
-			<?php
-			if (variable_get('user_relationships_show_user_pictures', 0))
-			{
-				$alt = t("@user's picture", array('@user' => $buddy->name ? $buddy->name : variable_get('anonymous', t('Anonymous'))));
-		  
-				$picture = theme('image', $buddy->picture, $alt, $alt, '', FALSE);
-				$attributes = array('attributes' => array('title' => t('View user profile.'), "class" => "thickbox"),
-									"query" => array("height" => 442,
-													"width" => 603,
-													"TB_iframe" => "true"),
-									'html' => TRUE);
-				print l($picture, "poker/profile/profile/$oBuddy->uid", $attributes);
-			}
-			?>
-		</div>
-		<div class="buddy_list_chips">
-			<?php print $buddy->Chips(TRUE); ?>
-		</div>
-		<?php if (!$hide_links) { ?>
-		<div class="buddy_list_links">
-			<a href="javascript:void(0);" onclick="javascript:os_poker_send_message({type :'os_poker_challenge_user', challengetarget: <?php print $buddy->uid; ?>});"><?php print t("Challenge"); ?>&nbsp;&gt;&gt;</a><br/>
-			<?php
-			
-				if ($buddy->Online())
-				{
-					$text = t("Join table");
-					$url = "os_poker_send_message({type :'os_poker_invite_user', target: " . $buddy->uid . ", online: true})";
-				}
-				else
-				{
-					$text = t("Invite now");
-					$url = "os_poker_send_message({type :'os_poker_invite_user', target: " . $buddy->uid . "})";
-				}
-			?>
-			<a href="javascript:void(0);" onclick="javascript:<?php print $url; ?>;"><?php print $text; ?>&nbsp;&gt;&gt;</a><br/>
-			<?php
-				if ($buddy->profile_accept_gifts == 0)
-				{
-					print l(t("Send Gifts") . "&nbsp;&gt;&gt;", "poker/shop/shop/1/buddy/" . $buddy->uid, array("query" => array("height" => 442,
-																																 "width" => 603,
-																																 "TB_iframe" => "true"),
-																												"attributes" => array("class" => "thickbox"),					
-																												"html" => TRUE));
-				}
-			?>
-		</div>
-		<?php } ?>
-	<?php } else { ?>
-		<div class="buddy_list_placeholder">
-			<div class="title">
-				<?php print t("Invite"); ?>
-			</div>
-			<div class="body">
-				<?php print l(t("Invite Friends"), "poker/buddies/invite", array("query" => array(	"height" => 442,
-																									"width" => 603,
-																									"TB_iframe" => "true"),
-																				"attributes" => array("class" => "thickbox"))); ?>
-			</div>
-		</div>
-	<?php } ?>
-</div>
diff -r 2feafdeba393 themes/poker/buddy_search.tpl.php
--- a/themes/poker/buddy_search.tpl.php	Tue Dec 29 08:33:09 2009 +0100
+++ /dev/null	Thu Jan 01 00:00:00 1970 +0000
@@ -1,3 +0,0 @@
-<div id="buddy_search">
-
-</div>
\ No newline at end of file
diff -r 2feafdeba393 themes/poker/error_message.tpl.php
--- a/themes/poker/error_message.tpl.php	Tue Dec 29 08:33:09 2009 +0100
+++ /dev/null	Thu Jan 01 00:00:00 1970 +0000
@@ -1,5 +0,0 @@
-<div class="poker_error_message">
-		<div class="msg">
-			<?php print $text; ?>
-		</div>
-</div>
\ No newline at end of file
diff -r 2feafdeba393 themes/poker/template.php
--- a/themes/poker/template.php	Tue Dec 29 08:33:09 2009 +0100
+++ b/themes/poker/template.php	Tue Dec 29 09:29:09 2009 +0100
@@ -1,53 +1,22 @@
 <?php
 // $Id: template.php $
 
-function poker_theme()
-{
+/**
+ * TODO: Is this needed. These are overrides for module's template. Adding
+ *       .tpl.php files should ne enough (unless they need to renamed but
+ *       it is probably not really needed).
+ */
+function poker_theme() {
   return array(
-				'user_login_block' => array(
-										'template' => 'user_login',
-										'arguments' => array('form' => NULL),
-									),
-									
-				'user_relationships' => array(
-											  'arguments' => array('account' => NULL, 'rtid' => NULL),
-											  'template'  => 'user_relationships',
-											  ),
-											  
-				'buddy_block' => array(
-										  'arguments' => array('buddy' => NULL, 'buddyNumber' => NULL, 'hide_links' => FALSE),
-										  'template'  => 'buddy_list_block',
-										  ),	
-										  
-				'buddies_tabs' => array(
-											  'arguments' => array('action' => NULL),
-											  'template'  => 'buddies_tabs',
-											  ),
-										  
-				'buddies_list' => array(
-											'arguments' => array('buddies' => NULL, 'current_user' => NULL, 'action' => NULL, 'page' => NULL),
-											'template'  => 'buddies_list',
-										),
-										
-				'buddies_invite' => array(
-										  'arguments' => array('form' => NULL, 'current_user' => NULL),
-										  'template'  => 'buddies_invite',
-										),
-										
-				'buddies_invitedlist' => array(
-										  'arguments' => array('current_user' => NULL),
-										  'template'  => 'buddies_invitedlist',
-										),
-										
-				'buddies_search' => array(
-							  'arguments' => array('form' => NULL, 'current_user' => NULL),
-										  'template'  => 'buddies_search',
-										),
-				'poker_error_message' => array(
-										  'arguments' => array('text' => NULL),
-										  'template'  => 'error_message',
-										),
-				);
+    'user_login_block' => array(
+      'template' => 'user_login',
+      'arguments' => array('form' => NULL),
+    ),
+    'user_relationships' => array(
+      'arguments' => array('account' => NULL, 'rtid' => NULL),
+      'template'  => 'user_relationships',
+    ),
+  );
 }
 
 function phptemplate_username($object) {
